### PR TITLE
add gpg check function

### PR DESCRIPTION
**Pull request by firecyberice**
connects to #44

### CODE REVIEW

File: builder/chroot-script.sh [hunk 1]
```diff
@@ -1,6 +1,90 @@
 #!/bin/bash
 set -ex
 
+KEYSERVER="ha.pool.sks-keyservers.net"
```
Comment (Govinda-Fichtner) [inline builder/chroot-script.sh]: @firecyberice Why is this code here as well as in gpgcheck.sh?

**Comment by Govinda-Fichtner** [comment]
@firecyberice Is PR considered finished?

**Comment by firecyberice** [comment]
I prefer to include the gpgcheck.sh file but as we redirect the chroot-script.sh into the chroot environment  the sourcing of the gpgcheck file does not work.

I added a return value in my gpg-check function maybe an exit is better in case on an error.

**Comment by StefanScherer** [comment]
@firecyberice so we can remove the gpgcheck.sh then?

**Comment by StefanScherer** [comment]
LGTM

**Pull request merged**